scripts/ci/release-latest

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Shouldn't this be updating the always open dcos-ui latest PR? But we never check out this branch on DC/OS as far as I can tell.

Reply:
not sure if I get you here: this script updates `buildinfo.json` in the DCOS built in `DCOS_BRANCH` variable.

Reply:
"jein" :) not that exact one, but there will be a new branch which is updated on every merge, see this line: https://github.com/dcos/dcos-ui/pull/2869/files#diff-03a0878332e0872cd3cacb5f10a1de09R33

The idea is to have multiple "latest" PRs for every DC/OS version, in theory we are able to have one for each DC/OS target version:
- `dcos-ui/master/dcos-ui-latest`
- `dcos-ui/1.12/dcos-ui-latest`
- `dcos-ui/1.11/dcos-ui-latest`
and so on…

we have to create a new PR for every release manually, but I think thats fine